frontend/public/components/modals/_modals.scss

```diff
@@ -19,14 +29,9 @@
   overflow-y: auto;
   padding: 0;
   position: relative;
-  @include scroll-shadows-vertical;
   -webkit-overflow-scrolling: touch;
 }
 
-.modal-body-border {
-  border-top: 1px solid $pf-color-black-300;
```

Comment:
The original operator install modal designs included a line between the heading and scrolling body.
https://github.com/itsptk/openshift-origin-design/blob/3cbbabfc06819bea6eadb05dc85e49d218ad2eab/designs/administrator/olm/select-install-operator-version/index.md

Reply:
@sg00dwin updated